util/measures/2022/quality-strata.csv

```diff
@@ -5,22 +5,24 @@ measureId,stratumName,shortDescription,longDescription,initialPopulationUuid,den
 239,nutrition,Patients who had counseling for nutrition during a visit that occurs during the measurement period,Patients who had counseling for nutrition during a visit that occurs during the measurement period,,,,,
 239,physicalActivity,Patients who had counseling for physical activity during a visit that occurs during the measurement period,Patients who had counseling for physical activity during a visit that occurs during the measurement period,,,,,
 305,14DaysOfDiagnosis,Patients who initiated treatment within 14 days of the diagnosis,Patients who initiated treatment within 14 days of the diagnosis,,,,,
 305,30DaysOfVisit,Patients who initiated treatment and who had two or more additional services with an AOD diagnosis within 30 days of the initiation visit,Patients who initiated treatment and who had two or more additional services with an AOD diagnosis within 30 days of the initiation visit,,,,,
 366,visitWithin30Days,Percentage of children who had one follow-up visit with a practitioner with prescribing authority during the 30-Day Initiation Phase.,Percentage of children who had one follow-up visit with a practitioner with prescribing authority during the 30-Day Initiation Phase.,,,,,
 366,overall,"Percentage of children who remained on ADHD medication for at least 210 days and who, in addition to the visit in the Initiation Phase, had at least two additional follow-up visits with a practitioner within 270 days (9 months) after the Initiation Phase ended.","Percentage of children who remained on ADHD medication for at least 210 days and who, in addition to the visit in the Initiation Phase, had at least two additional follow-up visits with a practitioner within 270 days (9 months) after the Initiation Phase ended.",,,,,
-370,12-17yrs,"Percentage of adolescent patients (aged 12-17 years) with a diagnosis of major depression or dysthymia and an
-initial PHQ-9 or PHQ-9M score greater than nine during the index event who reached remission at twelve
-months as demonstrated by a twelve month (+/-60 days) PHQ-9 or PHQ-9M score of less than 5.","Percentage of adolescent patients (aged 12-17 years) with a diagnosis of major depression or dysthymia and an
-initial PHQ-9 or PHQ-9M score greater than nine during the index event who reached remission at twelve
+370,12-17yrs,"Percentage of adolescent patients (aged 12-17 years) with a diagnosis of major depression or dysthymia and an
+initial PHQ-9 or PHQ-9M score greater than nine during the index event who reached remission at twelve
+months as demonstrated by a twelve month (+/-60 days) PHQ-9 or PHQ-9M score of less than 5.","Percentage of adolescent patients (aged 12-17 years) with a diagnosis of major depression or dysthymia and an
+initial PHQ-9 or PHQ-9M score greater than nine during the index event who reached remission at twelve

```

Comment:
The changes on these lines are meta data only. Due to the script change above removing Carriage Return characters, this meta data change does not affect the `measures-data`

Reply:
naughty excel adding whitespace